tests/test_biologic_reader.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
```

Comment:
I think there is something here, about the structure of tests vs. "module demo" which is what I would refer to this as.

I would like to reserve the "tests" folder for software tests run by pytest, and the data files to be used for the software tests (which should be kept as small as possible for small test run times) to be kept somewhere under the "tests" folder.

As for this module. If you intend it to be a demo for users to see, then I would suggest a "demo" folder under ixdat, so that they can be run programmatically from the installed package (and the data should then be kept somewhere where it can be included in the package). If, however, this serves merely as a the script that can be conveniently run during development (or further development) or the Reader, then I think it might as well live in the `if __name__ == "__main__"` part of the reader module.

Please note, that I know that you had some ideas about including data and demos and such and I don't want to interfere too much with that as I'm sure that will be sensible. I mainly just want to keep the tests folder for software tests.

Reply:
Yes, I intend it to be readily available for anyone who might want to develop this reader - and that is how I used it, running it again and again while writing the reader. I guess then if `__name__ == __main__` is a fine place. Will move after doing the rest of the stuff in this review.

Reply:
Shit, doesn't work. 
Are module demos incompatible with relative imports?
@KennethNielsen, I need your help before I can move forward on this. Trying to run the biologic module gives the following error:
```
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2020.3.2\plugins\python-ce\helpers\pydev\_pydev_bundle\pydev_umd.py", line 197, in runfile
    pydev_imports.execfile(filename, global_vars, local_vars)  # execute the script
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2020.3.2\plugins\python-ce\helpers\pydev\_pydev_imps\_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "C:/Users/scott/git/ixdat/src/ixdat/readers/biologic.py", line 7, in <module>
    from . import TECHNIQUE_CLASSES
  File "C:\Program Files\JetBrains\PyCharm Community Edition 2020.3.2\plugins\python-ce\helpers\pydev\_pydev_bundle\pydev_import_hook.py", line 21, in do_import
    module = self._system_import(name, *args, **kwargs)
ImportError: attempted relative import with no known parent package
```

Reply:
Moved instead to **ixdat/development_scripts**. Resolving.

Reply:
... and now, after our conversation on slack, moved back to the module under the 
`if __name__ == "__main__"` block. That block starts with this:
```
    """To run this module demo in PyCharm, configure it to run with 
    Module name = ixdat.readers.biologic,
    and *not* Script path = ...
    """
```